dietpi/dietpi-software

```diff
@@ -1959,6 +1941,31 @@ _EOF_
 		aSOFTWARE_ONLINEDOC_URL[$software_id]='https://dietpi.com/docs/software/desktop/#quiterss'
 		aSOFTWARE_REQUIRES_DESKTOP[$software_id]=1
 
+		# Browsers
+		#--------------------------------------------------------------------------------
+		software_id=113
+
+		aSOFTWARE_NAME[$software_id]='Chromium'
+		aSOFTWARE_DESC[$software_id]='web browser for desktop or autostart'
+		aSOFTWARE_CATEGORY_INDEX[$software_id]=29
+		aSOFTWARE_ONLINEDOC_URL[$software_id]='https://dietpi.com/docs/software/browsers/#chromium'
+		aSOFTWARE_REQUIRES[$software_id]='5 6'
+		#------------------
+		software_id=190
+
+		aSOFTWARE_NAME[$software_id]='Firefox'
```

Comment:
Thanks for that 👍, makes sense to have this as a separate option in any case. It should be even possible to install and run it in a dedicated X server session, without a wrapping desktop.